pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "ipywidgets>=8.0.6",
     "solara>=1.40.0",
     "pyyaml>=5.4.1",
-    "specutils>=1.16",
+    "specutils@git+https://github.com/astropy/specutils.git@main",  # TODO: >=1.18.0
```

Comment:
This requires Ricky to release specutils 0.18 first. We cannot merge this PR until that release happens.